### PR TITLE
zaz, fix recipe, cleanup

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Marked !x86_64 as it produces KDL's, ticket created at: https://dev.haiku-os.org/ticket/14479

File: games-puzzle/zaz/zaz-1.0.0.recipe [hunk 1]
```diff
@@ -9,19 +9,20 @@ other games of this type are that the balls have to be picked from the path \
 But one thing makes this game stand out most from it's predecessors - it's \
 free as in freedom ! You are free to download it, free to play it, free to \
 modify, love and hate it."
-HOMEPAGE="http://zaz.sourceforge.net" 
+HOMEPAGE="http://zaz.sourceforge.net"
+COPYRIGHT="2009-2010 Remigiusz Dybka, Leonard Ritter and Kinga Dybka"
+LICENSE="GNU GPL v3"
+REVISION="1"
 SOURCE_URI="http://sourceforge.net/projects/zaz/files/zaz-$portVersion.tar.bz2"
```
Comment (alaviss) [inline games-puzzle/zaz/zaz-1.0.0.recipe]:
```diff
-SOURCE_URI="http://sourceforge.net/projects/zaz/files/zaz-$portVersion.tar.bz2"
+SOURCE_URI="https://sourceforge.net/projects/zaz/files/zaz-$portVersion.tar.bz2"
```

File: games-puzzle/zaz/zaz-1.0.0.recipe [hunk 2]
```diff
@@ -48,42 +49,34 @@ BUILD_REQUIRES="
...
+#	Gettextize wants the user to press enter after running it, remove that
+#	"feature" from it...
 	grep -v '/dev/tty' < /bin/gettextize > gettextize
 	chmod +x gettextize
 
-#libtoolize --force --copy --install
-	automake --add-missing
-	aclocal
+	autoreconf -vfi
 	./gettextize -f
```
Comment (alaviss) [inline games-puzzle/zaz/zaz-1.0.0.recipe]: I wonder if this is still needed.

Comment (Begasus) [reply games-puzzle/zaz/zaz-1.0.0.recipe]: It is :) version mismatch if I leave it out

Comment (Begasus) [reply games-puzzle/zaz/zaz-1.0.0.recipe]: Still needed, without autoreconf it's using "-I/usr/include/freetype2" for freetype's headers

**Pull request merged**